ui/app/components/secret-engine/configure-azure.hbs

```diff
@@ -0,0 +1,107 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" (perform this.submitForm)}} aria-label="save azure creds" data-test-configure-form>
```

Comment:
I'm not sure if there are specific guidelines for `aria-label` text... but I think it should explain the form a bit more?
```suggestion
<form {{on "submit" (perform this.submitForm)}} aria-label="configure azure credentials" data-test-configure-form>
```